package.json

```diff
@@ -31,11 +31,13 @@
     "@angular/router": "^7.2.15",
     "@types/moment-timezone": "^0.5.12",
     "angular-froala-wysiwyg": "^3.0.0-rc.2-1",
+    "cookieconsent": "^3.1.1",
```

Comment:
Why is this needed ?

Reply:
This is one of the dependency package for `ngx-cookieconsent` as mentioned in the docs in [here](https://www.npmjs.com/package/ngx-cookieconsent)

Reply:
Okay.